_posts/2019-07-19-use-your-own-device-id.adoc

```diff
@@ -0,0 +1,38 @@
+= Configure your own device IDs
+:page-layout: page
+:page-categories: [client-config]
+:page-date: 2019-07-19 10:17:25
+:page-order: 99
+:icons: font
+
+In OTA Connect, a device has two types of identifier: an internal device UUID, and a standard device ID.
+
```

Comment:
...and the device_name.
When a user is looking at the UI, they can see 3 different identifiers, though I guess for simplicity you're ignoring the largest, most prominent one? :D

Reply:
...especially as there is no sign of the Device_ID when you're in the DDV. Only the internal uuid in the URL, and the device_name.